PocketKit/Sources/PocketKit/Collections/CollectionViewController.swift

```diff
@@ -170,6 +171,10 @@ class CollectionViewController: UIViewController {
 }
 
 private extension CollectionViewController {
+    enum Constants {
+        static let offlineSectionHeight: CGFloat = 0.65
```

Comment:
can we add a comment to why we are using this specific value?

Reply:
yes we can!